packages/wrangler/src/index.tsx

```diff
@@ -501,6 +511,48 @@ export function createCLIParser(argv: string[]) {
 		}
 	);
 
+	wrangler.command(
+		"deployments",
+		"🚢 Logs the 10 most recent deployments with 'Version ID', 'Version number','Author name', and 'Latest deploy'",
+
+		(yargs) => {
+			yargs.option("name", {
+				describe: "The name of your worker",
+				type: "string",
+			});
+		},
+		async (deploymentsYargs: ArgumentsCamelCase<{ name: string }>) => {
+			await printWranglerBanner();
+			const config = readConfig(
+				deploymentsYargs.config as ConfigPath,
+				deploymentsYargs
+			);
+			const accountId = await requireAuth(config);
+			const scriptName = getScriptName(
+				{ name: deploymentsYargs.name, env: undefined },
+				config
+			);
+			const scriptMetadata = await fetchResult<ServiceMetadataRes>(
+				`/accounts/${accountId}/workers/services/${scriptName}`
+			);
+
+			const scriptTag = scriptMetadata.default_environment.script.tag;
+			const deployments = await fetchResult<DeploymentListRes>(
+				`/accounts/${accountId}/workers/versions/by-script/${scriptTag}`
+			);
+
+			const versionMessages = deployments.versions.map(
```

Comment:
Do we want to maybe include the date in the message? Field is `versions.metadata.created_on`.

Reply:
I dont know maybe Liz has input on this @lrapoport-cf

Reply:
@danielrs i suggest we leave it out for now so we can get the MVP of this feature landed, and i'll add a ticket to include it for later. does that work?

Reply:
I'm fine with either, it is a quick addition.

Reply:
following internal discussion, let's add the date now 👍